lib/matplotlib/tests/test_texmanager.py

```diff
@@ -42,3 +42,12 @@ def test_font_selection(rc, preamble, family):
     src = Path(tm.make_tex("hello, world", fontsize=12)).read_text()
     assert preamble in src
     assert [*re.findall(r"\\\w+family", src)] == [family]
+
+def test_usetex_with_underscore():
```

Comment:
This probably should go in `test_usetex.py` which already has a guard at the top to skip the tests when latex is not available.